### DISCUSSION TITLE
Using lego with --domains*."<website.com>" and get back flag provided but not defined

### DISCUSSION THREAD

**Discussion by lionheartdevs** [go-acme/lego#1742] [Q&A] [answered]
using  /letsencript/lego and registering a domain for a wildcard SSL and the issue with comes back is: Incorrect Usage. flag provided but not defined: -domains*.<domainhere>

The code used was:  --email="<emailhere>" --domains="<domainhere>" --domains"*.<domainhere>" --dns <providerhere> --path="<pathhere>" run - but it comes back with -domains*.<domainhere>

is --domains"*. not valid for wildcards anymore?

Thanks.

**Comment by ldez** [comment, marked as answer]
hello,

The problem is related to the missing `=` between the flag and the value.

**Reply by lionheartdevs** [reply]
Arg..I can't believe I missed that....thank you so much! lol
